controllers/admin/AdminOrdersController.php

```diff
@@ -2078,7 +2078,15 @@ public function ajaxProcessSendMailValidateOrder()
         }
     }
 
-    public function ajaxProcessAddProductOnOrder()
+    /**
+     * Add product on an order
+     *
+     * @throws PrestaShopDatabaseException
```

Comment:
This method doesn't throws anything :thinking: 
If you want to add all possibilities, it will be really really hard to maintain

Reply:
Fixed